docs/getting_started/discover_ui.md

```diff
@@ -14,15 +14,20 @@ Depending on your location within the Back Office, it can contain the following
 - Global search field that allows you to [find content](../search/search_for_content.md) by checking all searchable Fields
 - Drop-down that changes the site context
 - Notification icon that informs you about incoming assignments, for example, items assigned for [your review](../content_management/workflow_management/editorial_workflow.md#review-queue)
+- App switcher which includes links to official websites of the QNTM group ecosystem companies, world-class ad-tech and mar-tech solution providers
```

Comment:
```suggestion
- App switcher that includes links to official websites of the QNTM group companies, including world-class ad-tech and mar-tech solution providers
```

Reply:
Can we use different verb in the second part, because currently we repeat "includes"... "including" :)

Reply:
Let's leave it with the first version, without this suggestion :)